### PR TITLE
:bar_chart: guinea worm data update 2023 numbers

### PR DESCRIPTION
**Pull request by antea04**
PR to update the guinea worm case dataset with the latest values (owid/owid-issues#1402)

Tasks:
- [x] create a new fasttrack sheet for case data including 2023
- [x] combine with certification status data & previous case data (already in etl)
- [x] update charts

Choices: 
- added an extra column in grapher step to get categorical/ ordinal values for year of certification/ certification status
- added metadata in grapher step (that is why it throws warnings before that)
  - added origins to each variable since fast-track adds only sources

### CODE REVIEW

File: etl/steps/data/garden/who/2024-06-17/guinea_worm.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+
+def add_current_year(tb, tb_cases, year, changes_dict={}):
+    """
+    Add rows with current certification status & case numbers for each country
+    tb (Table): table with certification status & case numbers until last year
+    tb_cases (Table): table with case numbers for all (incl. current) years
+    year (int): current year
+    changes_dict (dict): changes to certification status since last year with key: country, value: certification status
+    (changes_dict is empty for 2023, including it to make code reusable)
```
Comment (spoonerf) [inline etl/steps/data/garden/who/2024-06-17/guinea_worm.py:82]: Could you briefly describe how you would use it?

File: etl/steps/data/grapher/who/2024-06-17/guinea_worm.meta.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          - "2000s"
+          - "2010s"
+          - "2020s"
+          - "Pre-certification"
+          - "Endemic"
+      certification_status:
+        title: Certification status over time
+        unit: ""
+        description_short:
+          The current and historical values for the status of Guinea worm disease (Dracunculiasis) as certified by the WHO.
```
Comment (spoonerf) [inline etl/steps/data/grapher/who/2024-06-17/guinea_worm.meta.yml:71]: You could add the DoD in here e.g. [Guinea worm disease (Dracunculiasis)](#dod:guinea-worm)

File: etl/steps/data/grapher/who/2024-06-17/guinea_worm.meta.override.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  guinea_worm:
+    variables:
+      time_frame_certified:
+        description_key: []
+        description_processing: |-
+          The current and historical values for the status of Guinea worm disease (Dracunculiasis) as certified by the WHO. To be certified as free of guinea worm disease, a country must have reported zero indigenous cases through active surveillance for at least three consecutive years.
+          Data regarding certification status is taken from: https://web.archive.org/web/20211024081702/https://apps.who.int/dracunculiasis/dradata/html/report_Countries_t0.html
+
+          This is supplemented with more recent changes to Guinea worm disease certification:
+          - Angola has had endemic status since 2020:  https://www.who.int/news/item/23-09-2020-eradicating-dracunculiasis-human-cases-and-animal-infections-decline-as-angola-becomes-endemic
```
Comment (spoonerf) [inline etl/steps/data/grapher/who/2024-06-17/guinea_worm.meta.override.yml:11]: You could be a little bit more explicit that we have actively added this information to the dataset (some discussion of the type of voice we should use in `description_processing` [here.](https://owid.slack.com/archives/C03NA1B2P1U/p1709129035048969)

**Review by spoonerf — APPROVED** [review]
Very nice! I've added a few comments, but nothing major. A couple of extra comments about the charts:

* Could you add this to charts that mention 'Pre-certification' where it isn't already in the footnote or subtitle:

> "Countries labeled pre-certification are endemic countries that have reported zero indigenous cases in the calendar year."

* Could you also set the source to default in the chart admin, so that the year is added?

* The chart comparing data from the Carter Centre and the WHO can be deleted (I think I already did this on live, but if it reappears can you delete it again?

File: etl/steps/data/garden/who/2024-06-17/guinea_worm.py [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    tb_cases = snap.read_csv()
+    tb_cert = ds_garden["guinea_worm"].reset_index()
+
+    #
+    # Process data.
+    #
+
+    # harmonize countries
+    tb_cert = geo.harmonize_countries(df=tb_cert, countries_file=paths.country_mapping_path)
+    tb_cases = geo.harmonize_countries(df=tb_cases, countries_file=paths.country_mapping_path)
```
[the remark this answers is not in the document]
Comment (antea04) [reply etl/steps/data/garden/who/2024-06-17/guinea_worm.py:32]: ah, thats handy - thanks!

**Pull request merged**